src/Symfony/Bundle/SecurityBundle/Resources/views/Collector/security.html.twig

```diff
@@ -120,6 +126,19 @@
         </div>
     {% endif %}
 
+    {% if collector.firewall|default([]) is not empty %}
+        <h2>Security Firewall</h2>
+
+        <div class="metrics">
+            <div class="metric">
+                <span class="value">{{ collector.firewall.name|default('unknown') }}</span>
```

Comment:
What about `n/a` instead of `unknown`? Should it always have a name as config key anyway?